wpilibc/src/main/native/include/Commands/Command.h

```diff
@@ -117,42 +112,41 @@ class Command : public ErrorBase, public NamedSendable {
 
  private:
   void LockChanges();
-  /*synchronized*/ void Removed();
+  void Removed();
   void StartRunning();
   void StartTiming();
 
-  /** The name of this command */
+  // The name of this command
```

Comment:
Are we making these non-doxygen comments because they're private?

Reply:
Private variables don't show up in Doxygen. Public and protected ones do.